ccd-definition/ComplexTypes/CareSupervision/1_SupportingDocuments/SupplementsBundle.json

```diff
@@ -0,0 +1,52 @@
+[
```

Comment:
it should be rather Supplement and not SupplementBundle, it keep only one document and not bundle. It would make consistent with java code as well

Reply:
Renamed the complex type